ner_v2/api.py

```diff
@@ -243,3 +246,92 @@ def number(request):
         return HttpResponse(status=500)
 
     return HttpResponse(json.dumps({'data': entity_output}), content_type='application/json')
+
+
+def phone_number(request):
+    """Uses PhoneDetector to detect phone numbers
+
+        request params:
+            message (str): natural text on which detection logic is to be run. Note if structured value is
+                                   detection is run on structured value instead of message
+            entity_name (str): name of the entity. Also acts as elastic-search dictionary name
+                              if entity uses elastic-search lookup
+            structured_value (str): Value obtained from any structured elements. Note if structured value is
+                                   detection is run on structured value instead of message
+                                   (For example, UI elements like form, payload, etc)
+            fallback_value (str): If the detection logic fails to detect any value either from structured_value
+                             or message then we return a fallback_value as an output.
+            bot_message (str): previous message from a bot/agent.
+            source_language (str): language for which the phone numbers have to be detected
+
+        Returns:
+            response (django.http.response.HttpResponse): HttpResponse object
+        Examples:
+
+        message = "Call 02226129857 and message +1(408) 92-124 and send 100rs to 91 9820334416 9920441344"
+        entity_name = 'number_of_unit'
+        structured_value = None
+        fallback_value = None
+        bot_message = None
+        source_language = 'en'
+
+        output:
+
+        {
+    "data": [
```

Comment:
Where is 'data' key added? I think key should be entity name.

Reply:
Done